kernel/base/src/main/java/com/twosigma/beakerx/widgets/Widget.java

```diff
@@ -42,11 +42,11 @@
   
   public enum CommActions {
 
-    DOUBLE_CLICK("DOUBLE_CLICK"),
+    DOUBLE_CLICK("ondoubleclick"),
```

Comment:
we need to keep this i think to maintain this fix as well:
https://github.com/twosigma/beakerx/issues/5536